proto/prehog/v1alpha/teleport.proto

```diff
@@ -138,13 +138,6 @@ enum DiscoverResource {
   DISCOVER_RESOURCE_APPLICATION_HTTP = 8;
   DISCOVER_RESOURCE_APPLICATION_TCP = 9;
   DISCOVER_RESOURCE_WINDOWS_DESKTOP = 10;
-  DISCOVER_RESOURCE_DATABASE_SQLSERVER_RDS = 11;
-  DISCOVER_RESOURCE_DATABASE_POSTGRES_REDSHIFT = 12;
-  DISCOVER_RESOURCE_DATABASE_SQLSERVER_SELF_HOSTED = 13;
-  DISCOVER_RESOURCE_DATABASE_REDIS_SELF_HOSTED = 14;
-  DISCOVER_RESOURCE_DATABASE_POSTGRES_GCP = 15;
-  DISCOVER_RESOURCE_DATABASE_MYSQL_GCP = 16;
-  DISCOVER_RESOURCE_DATABASE_SQLSERVER_GCP = 17;
```

Comment:
Should these be removed?

Reply:
Definitely not,  I reset the proto again!